src/main/java/org/opentripplanner/routing/api/request/refactor/request/ViaLocation.java

```diff
@@ -0,0 +1,23 @@
+package org.opentripplanner.routing.api.request.refactor.request;
+
+import java.time.Duration;
+import org.opentripplanner.model.GenericLocation;
+
+public class ViaLocation {
+
+  private GenericLocation point;
+  private boolean passThroughPoint = false;
+  private Duration slack = Duration.ofMinutes(60);
```

Comment:
Is this minimum or maximum slack?

Reply:
Not sure. This class is not used anywhere yet. It will be needed when we implement via searches. Guess we change change it then?